recipe/meta.yaml

```diff
@@ -20,19 +20,21 @@ source:
     - osx.patch  # [osx]
 
 build:
-  number: 0
+  number: 1
   detect_binary_files_with_prefix: true
   skip: true  # [win]
 
 requirements:
   build:
     - gcc  # [osx]
+    - libgcc  # [osx]
```

Comment:
libgcc is potentially newer (could come from the GCC 5.2.0 docker image I made last year).  Installing them both at once is probably not a great idea - it's then ambiguous which files are present.  Whatever package got installed last...